src/btree/bt_debug.c

```diff
@@ -680,22 +659,16 @@ __debug_cell_kv(
     case WT_CELL_VALUE_OVFL_RM:
     case WT_CELL_VALUE_SHORT:
         if (!WT_TIME_WINDOW_IS_EMPTY(&unpack->tw))
-            WT_RET(ds->f(ds, ", %s", __wt_time_window_to_string(&unpack->tw, time_string)));
+            WT_RET(ds->f(ds, " | %s", __wt_time_window_to_string(&unpack->tw, time_string)));
         break;
     }
 
-    /* Column-store deleted cells. */
```

Comment:
Do we no longer print deleted cells? I don't see this code moved elsewhere

Reply:
it's calling on `__wt_cell_type_string` to print the cell type and we already print that info towards the top of the function - so a bit redundant. Spotted thanks to @Siddhartha8899